x/oracle/types/genesis.go

```diff
@@ -0,0 +1,49 @@
+package types
+
+import (
+	"encoding/json"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// DefaultGenesis returns the default Capability genesis state
+func DefaultGenesis() *GenesisState {
+	return &GenesisState{
+		Oracles:             []Oracle{},
+		OracleRegistrations: []OracleRegistration{},
+		Params:              DefaultParams(),
+	}
+}
+
+func GetGenesisStateFromAppState(cdc codec.Codec, appState map[string]json.RawMessage) *GenesisState {
```

Comment:
I have a question. Is this function neccessary?

Reply:
This is the function used in the command to add genesis oracle. It's not important right now, but I think it will be used later.